server/base/src/test/java/org/apache/accumulo/server/conf/NamespaceConfigurationTest.java

```diff
@@ -114,6 +115,7 @@ public void testGet_InParent() {
     assertEquals("sekrit", nsConfig.get(Property.INSTANCE_SECRET));
 
     // TODO Need to check parent and accessor usage
+    verify(parent, propStore, context);
```

Comment:
@EdColeman This is failing but I wasn't sure what you meant by the TODO. Suggestions?